spec.md

```diff
@@ -1657,40 +1630,37 @@ If a `CreateSnapshot` operation times out before the snapshot is cut, leaving th
 The purpose of `freeze` is to ensure the application data is in consistent state.
 When `freeze` is performed, the container is paused and the application is also paused.
 When `thaw` is performed, the container and the application start running again.
-During the snapshot uploading phase, since the snapshot is already cut, a `thaw` operation can be performed so application can start running without waiting for the upload to complete.
-The status of the snapshot will become `READY` after the upload is complete.
-
-For cloud providers and storage systems that don't have the uploading process, the status should be `READY` after the snapshot is cut.
-`thaw` can be done when the status is `READY` in this case.
+During the snapshot processing phase, since the snapshot is already cut, a `thaw` operation can be performed so application can start running without waiting for the process to complete.
+The `is_ready_to_use` parameter of the snapshot will become `true` after the process is complete.
 
-A `CREATING` status is not included here because CreateSnapshot is synchronous and will block until the snapshot is cut.
+For cloud providers and storage systems that don't have the process, the `is_ready_to_use` parameter should be `true` after the snapshot is cut.
+`thaw` can be done when the `is_ready_to_use` parameter is `true` in this case.
 
-`ERROR` is a terminal snapshot status.
-A CO SHOULD explicitly delete snapshots in this status.
+A gRPC error code SHALL be returned if an error occurs during any stage of the snapshotting process.
+A CO SHOULD explicitly delete snapshots when an error occurs.
 
-The SnapshotStatus parameter provides guidance to the CO on what action can be taken in the process of snapshotting.
+The `is_ready_to_use` parameter provides guidance to the CO on what action can be taken in the process of snapshotting.
 Based on this information, CO can issue repeated (idemponent) calls to CreateSnapshot, monitor the response, and make decisions.
 Note that CreateSnapshot is a synchronous call and it must block until the snapshot is cut.
-If the cloud provider or storage system does not need to upload the snapshot after it is cut, the status returned by CreateSnapshot SHALL be `READY`.
-If the cloud provider or storage system needs to upload the snapshot after the snapshot is cut, the status returned by CreateSnapshot SHALL be `UPLOADING`.
-CO MAY continue to call CreateSnapshot while waiting for the upload to complete until the status becomes `READY`.
+If the cloud provider or storage system does not need to process the snapshot (i.e., upload the snapshot) after it is cut, the `is_ready_to_use` parameter returned by CreateSnapshot SHALL be `true`.
+If the cloud provider or storage system needs to process the snapshot after the snapshot is cut, the `is_ready_to_use` parameter returned by CreateSnapshot SHALL be `false`.
```

Comment:
Move this line to right after lines 1646-1647

Reply:
Done